static-site/package.json

```diff
@@ -12,6 +12,7 @@
     "test": "gatsby build",
     "build:pp": "gatsby build --prefix-paths",
     "clean": "rm -rf public",
+    "lint": "DEBUG=eslint:cli-engine eslint --ext .js,.jsx .",
```

Comment:
Did you mean to leave this DEBUG variable set? I quite like the output it gives, so I'm happy for it to stay.

Reply:
Yes, I added it when investigating the discrepancy between local and GitHub runner ESLint runs. It's not overly verbose, so I don't see any harm in keeping it around.